app/controllers/orders/pending.js

```diff
@@ -15,10 +16,23 @@ export default Controller.extend({
     return this.get('model.order.paymentMode') === 'paypal';
   }),
 
+  isOmise: computed('model.order', function() {
+    return this.get('model.order.paymentMode') === 'omise';
+  }),
+
   paymentAmount: computed('model.order', function() {
     return this.get('model.order.amount') * 100;
   }),
 
+  publicKeyOmise: computed('settings.omiseLivePublic', function() {
+    return this.get('settings.omiseLivePublic') || this.get('settings.omiseTestPublic');
+  }),
+
+  omiseFormAction: computed('model.order', function() {
```

Comment:
Be specific when watching other properties. If you're using `model.order.identifier` then put that in the computed property.